core/templates/dev/head/app.js

```diff
@@ -43,6 +43,9 @@ oppia.constant('OBJECT_EDITOR_URL_PREFIX', '/object_editor_template/');
 // Feature still in development.
 // NOTE TO DEVELOPERS: This should be synchronized with the value in feconf.
 oppia.constant('ENABLE_ML_CLASSIFIERS', false);
+// NOTE: To use TRAINING_DATA_CLASSIFICATION, ENABLE_ML_CLASSIFIERS must be
+// True.
+oppia.constant('ENABLE_TRAINING_DATA_CLASSIFICATION', false);
```

Comment:
Why do we need a separate constant for this? Couldn't we just enable it automatically if ML classification is enabled?

Reply:
There is no need. I added it if we want to enable/disable training data classification separately from ML classification.